src/main/fc/config.h

```diff
@@ -72,6 +72,7 @@ typedef struct systemConfig_s {
     uint8_t activeRateProfile;
     uint8_t debug_mode;
     uint8_t task_statistics;
+    uint8_t cpu_overclock;
```

Comment:
Fair enough. `bool` seems to be used in other (newer?) structs used for PGs. Also, don't know if saving space here is the right way to go, since the data structures used in PGs will probably become part of the configurator API at some stage.